modules/dev-server/main.tf

```diff
@@ -5,6 +5,8 @@ locals {
     container_name   = "consul-server"
     container_port   = 8500
   }] : []
+
+  consul_license_enabled = var.consul_license != ""
```

Comment:
Would it make sense to call this `consul_enterprise` or `consul_enterprise_enabled`? I'm not sure if this will get re-used anywhere else in the future so maybe it's not worth it.